app/src/components/dashboard/services/Modal.js

```diff
@@ -9,64 +9,59 @@ export default function Modal(props) {
         MAIS INFORMACOES
       </p>
       <div className="flex p-10 justify-center">
-        <div className="text-center">
-          <p className="text-9xl mr-10 text-custom-lightBrown playfair">1</p>
-        </div>
         <div>
-          <div className="ml-20">
-            <div className="max-w-5xl m-auto">
-              {props.step1 || props.step2 || props.step3 ? (
-                <div className="flex items-center text-start pb-4">
-                  <Number step={"1"} />
+          <div className="max-w-5xl m-auto">
+            {props.step1 || props.step2 || props.step3 ? (
```

Comment:
why do you need this comparison if you are then checking if this props exists below?

Reply:
well noted, correction done!